minecraft_servers/endercraft/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+    "server_name": "endercraft",
+    "nice_name": "EnderCraft",
+    "direct_ip": "endercraft.com.br",
+    "server_wildcards": [
+      "%.endercraft.com.br"
+    ],
+    "supported_languages": [
+       "pt"
+    ],
+    "social": {
+      "web": "https://ecbr.me/site",
+      "web_shop": "https://ecbr.me/site",
+      "web_support": "https://ecbr.me/disc",
```

Comment:
web_support can't be Discord link, add FAQ or something like that; if non-existant, remove please